crates/db_schema/src/impls/post.rs

```diff
@@ -111,14 +111,13 @@ impl Post {
   ) -> Result<Vec<Self>, Error> {
     let conn = &mut get_conn(pool).await?;
 
-    let perma_deleted = "*Permananently Deleted*";
     let perma_deleted_url = "https://deleted.com";
```

Comment:
This feels off. If there must be a URL I'd prefer it being something along the lines of the instance's homepage rather than a hardcoded domain controlled by no-idea-who

Reply:
updated